### PR TITLE
R updates

### PR DESCRIPTION
**Pull request by smk0033**
- moved a notebook to new R section
- adjusted wording in rst file (per discussion during our R Doc meeting)

### CODE REVIEW

**Review by wildintellect — APPROVED** [review]
We probably need another doc on installing packages with R the conda way...

File: docs/source/technical_tutorials.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@ Technical Tutorials
 
 This section of the documentation includes hands-on tutorials with example algorithms and data-sets. It is focused on learning how to use the MAAP platform, and not science-oriented.
 
+**Note:** Tutorials are written in Python *except* for those under :ref:`Working with R <working_with_r_section>`. For current documents written in R, please see :ref:`Working with R <working_with_r_section>`.
```
Comment (zacdezgeo) [inline docs/source/technical_tutorials.rst:6]: Love the note at the top-level to provide more guidance. I added a link to the working with R section so a user can directly jump to the relevant content in this case!

File: docs/source/technical_tutorials/working_with_r.rst [hunk 2]
```diff
@@ -1,8 +1,11 @@
+.. _working_with_r_section:
```
Comment (zacdezgeo) [inline docs/source/technical_tutorials/working_with_r.rst:1]: I had to add the reStructuredText label (reference target) to create a unique identifier we could reference in the technical tutorial rst file.

**Review by zacdezgeo — APPROVED** [review]
I like the note you added on the top section of the technical tutorials. It helps R end-users imm

**Comment by smk0033** [comment]
Thanks, Zac!

Also added the conda thing to the R EPIC.

**Pull request merged**